styles.css

```diff
@@ -292,3 +323,74 @@ body {
   background: #febf52;
   background-image: url(assets/Shapes/3.3.svg);
 }
+
+/* ==================== Contact styles ==================== */
+
+.contact {
+  width: 100%;
+  margin-top: 27px;
+  background: url(Assets/Shapes/1.svg) #f9f9fa;
+  background-repeat: no-repeat;
+  padding: 106px 13px 31px 17px;
+  gap: 14px;
+  align-items: center;
+}
+
+.contact-info {
+  width: 100%;
+  max-width: 500px;
+  gap: 20px;
+}
+
+.contact-form {
+  width: 100%;
+  max-width: 500px;
+  gap: 20px;
+}
+
+.contact-form ul {
+  list-style: none;
+  gap: 10px;
+}
+
```

Comment:
Hi @Lizdev-05  👋 

Thank you so much for the good review ❤️, but it looks like you missed it.
as you can see I used Flexbox to lay out the elements in the contact section but I used a different class for that.
(it was a recommendation from a previous reviewer to create classes for Flexbox to avoid repetition).

`index.html` :
![image](https://user-images.githubusercontent.com/77485409/230349152-15d86572-6597-4be3-97c1-7a6cd695eba0.png)
`styles.css` :
![image](https://user-images.githubusercontent.com/77485409/230349430-72c07cbb-98fb-48cf-b244-b7b9ef45eeb1.png)

Reply:
Hi @chawkimbarki, Good job using classes to make your code DRY. This is a very good practice.

Reply:
Oops, I am so sorry about that error. Kindly resubmit. 🥺 🙏